modules/github-repository/locals.tf

```diff
@@ -105,3 +101,9 @@ locals {
 
   secrets = merge(local.plaintext_secrets, local.encrypted_secrets)
 }
+
+locals {
```

Comment:
please merge all locals{} blocks into one single locals{} block

Reply:
Maybe It would be good if we will separate locals by their purpose of use